api/app.py

```diff
@@ -38,19 +34,48 @@ def get_algorithms():
 @app.route('/analyze', methods=['POST'])
 @cross_origin(origin=['*'],supports_credentials=True)
 def analyze():
-    # export platform info to prometheus
-    response = requests.post(qujata_platform_exporter_target + "/export-platform-info")
-
     global test_is_running
-    print('analysis route was triggered.')
-    start_time = int(datetime.timestamp(datetime.now() - timedelta(seconds=60)) * 1000)
     data = request.get_json()
-    print(test_is_running)
+
+    __export_platform_data()
+    error = __validate(data)
+    if error != None:
+        return error
+
+    test_is_running = True
+    start_time = int(datetime.timestamp(datetime.now() - timedelta(seconds=60)) * 1000)
+    error = __start_analyze(data)
+    if error != None:
+        return error
+    end_time = int(datetime.timestamp(datetime.now() + timedelta(seconds=90)) * 1000)
```

Comment:
what is the reason we're adding **90** seconds specifically? and **60** on the start time?